<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1126</width>
    <height>800</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1126</width>
    <height>800</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string notr="true">Stock problem</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(3, 4, 19)</string>
  </property>
  <widget class="QPushButton" name="RunBtn">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>540</y>
     <width>241</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	border-radius: 20px;
	background-color:  #252984;
	color: white;
	font: bold 25px;
}
QPushButton:hover {
	border-radius: 20px;
	background-color:  #393c87;
	color: white;
	font: bold 25px;
}</string>
   </property>
   <property name="text">
    <string>Run</string>
   </property>
  </widget>
  <widget class="QPushButton" name="StepBtn">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>600</y>
     <width>241</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
border-radius: 20px;
background-color:  #252984;
color: white;
font: bold 25px;
}

QPushButton::flat {
border-radius: 20px;
background-color:  #5E5F70;
color: white;
font: bold 25px;
}

QPushButton:hover {
	border-radius: 20px;
	background-color:  #393c87;
	color: white;
	font: bold 25px;
}</string>
   </property>
   <property name="text">
    <string>Step</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="StepBox">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>650</y>
     <width>241</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QCheckBox {
	font: bold 20px;
	color: white;
}

QCheckBox:hover {
	font: bold 20px;
	color: #85ab85;
}

QCheckBox::indicator:unchecked {
	background-color: white;
	border: 0px solid black;
	border-radius: 3px;
}
QCheckBox::indicator:checked {
	background-color: green;
	border: 0px solid black;
	border-radius: 3px;
}</string>
   </property>
   <property name="text">
    <string>Step by step</string>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="tristate">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>710</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QProgressBar 
{
	background: #D9D9D9;
}
QProgressBar::chunk
{
    background: #31922F;
}</string>
   </property>
   <property name="value">
    <number>0</number>
   </property>
   <property name="textVisible">
    <bool>false</bool>
   </property>
   <property name="format">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>530</y>
     <width>801</width>
     <height>251</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="LogLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <item>
     <widget class="QLabel" name="LogTitle">
      <property name="styleSheet">
       <string notr="true">QLabel 
{
	padding-left: 20px;
	background: #1D253D;
	color: white;
	font: bold 16px;
}</string>
      </property>
      <property name="text">
       <string>Log details</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPlainTextEdit" name="LogConsole">
      <property name="styleSheet">
       <string notr="true">QPlainTextEdit 
{
	background: #121828;
	border: none;
	color: white;
}

QScrollBar:vertical
{
	background: #1D253D;
	border: 0px solid red;
	border-radius: 7px;
}

QScrollBar::handle:vertical
{
	background: #C6C8CE;
	border: 0px solid red;
	border-radius: 7px;
}
QScrollBar::add-line:vertical,
QScrollBar::sub-line:vertical,
QScrollBar::add-page:vertical,
QScrollBar::sub-page:vertical
{
	height: 0px;
}
</string>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAsNeeded</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="sizeAdjustPolicy">
       <enum>QAbstractScrollArea::AdjustToContents</enum>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="plainText">
       <string/>
      </property>
      <property name="backgroundVisible">
       <bool>false</bool>
      </property>
      <property name="centerOnScroll">
       <bool>false</bool>
      </property>
      <property name="placeholderText">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="ExitBtn">
   <property name="geometry">
    <rect>
     <x>1080</x>
     <y>10</y>
     <width>31</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">*{
	color: white;
	background: rgb(3, 4, 19);
	border: 0px solid rgb(3, 4, 19);;
}</string>
   </property>
   <property name="text">
    <string>X</string>
   </property>
  </widget>
  <widget class="QPushButton" name="TrayBtn">
   <property name="geometry">
    <rect>
     <x>1040</x>
     <y>10</y>
     <width>31</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">*{
	color: white;
	background: rgb(3, 4, 19);
	border: 0px solid rgb(3, 4, 19);;
}</string>
   </property>
   <property name="text">
    <string>_</string>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>19</x>
     <y>50</y>
     <width>1071</width>
     <height>441</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout" stretch="803,241">
    <property name="spacing">
     <number>35</number>
    </property>
    <item>
     <widget class="QGraphicsView" name="Canvas">
      <property name="styleSheet">
       <string notr="true">QGraphicsView 
{
	background: #D9D9D9;
}</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <item>
       <layout class="QVBoxLayout" name="PopulationLayout" stretch="20,197">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="PopulationLabel">
          <property name="styleSheet">
           <string notr="true">QLabel 
{
	padding-left: 10px;
	background: #1D253D;
	color: white;
	font: bold 16px;
}</string>
          </property>
          <property name="text">
           <string>Population</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QScrollArea" name="PopulationBox">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="MinimumExpanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>197</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>197</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">
	background: #121828;
	border: none;
</string>
          </property>
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents_3">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>235</width>
             <height>197</height>
            </rect>
           </property>
          </widget>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="VariablesLayout">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="VariablesLabel">
          <property name="styleSheet">
           <string notr="true">QLabel 
{
	padding-left: 10px;
	background: #1D253D;
	color: white;
	font: bold 16px;
}</string>
          </property>
          <property name="text">
           <string>Variables</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QScrollArea" name="VariablesBox">
          <property name="styleSheet">
           <string notr="true">
	background: #121828;
	border: none;
</string>
          </property>
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents_2">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>235</width>
             <height>192</height>
            </rect>
           </property>
          </widget>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="ImportBtn">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
border: 1px solid black;
background: #1D253D;
color: white;
font: bold 15px;
}
QPushButton:hover{
border: 1px solid black;
background: #262d42;
color: white;
font: bold 15px;
}</string>
   </property>
   <property name="text">
    <string>Import</string>
   </property>
  </widget>
  <widget class="QPushButton" name="HelpBtn">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>0</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
border: 1px solid black;
background: #1D253D;
color: white;
font: bold 15px;
}

QPushButton:hover{
border: 1px solid black;
background: #262d42;
color: white;
font: bold 15px;
}</string>
   </property>
   <property name="text">
    <string>Help</string>
   </property>
  </widget>
  <widget class="QComboBox" name="AlgVersBox">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>6</y>
     <width>72</width>
     <height>24</height>
    </rect>
   </property>
   <property name="palette">
    <palette>
     <active>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Button">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>29</red>
         <green>37</green>
         <blue>61</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Text">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="ButtonText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>29</red>
         <green>37</green>
         <blue>61</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Window">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>29</red>
         <green>37</green>
         <blue>61</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="PlaceholderText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Button">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>29</red>
         <green>37</green>
         <blue>61</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Text">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="ButtonText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>29</red>
         <green>37</green>
         <blue>61</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Window">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>29</red>
         <green>37</green>
         <blue>61</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="PlaceholderText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Button">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>29</red>
         <green>37</green>
         <blue>61</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Text">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="ButtonText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>29</red>
         <green>37</green>
         <blue>61</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Window">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>29</red>
         <green>37</green>
         <blue>61</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="PlaceholderText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
     </disabled>
    </palette>
   </property>
   <property name="styleSheet">
    <string notr="true">background: #1d253d ;
color: white;
font:  semibold 10px;</string>
   </property>
   <item>
    <property name="text">
     <string>Canonical</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Genitor</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Island</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="MutBtn">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>0</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
border: 1px solid black;
background: #1D253D;
color: white;
font: bold 15px;
}

QPushButton:hover{
border: 1px solid black;
background: #262d42;
color: white;
font: bold 15px;
}</string>
   </property>
   <property name="text">
    <string>Mutation</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
